code/datums/ruins.dm

```diff
@@ -0,0 +1,21 @@
+/datum/map_template/ruin
+	name = "A Chest of Doubloons"
```

Comment:
Is this used anywhere other than https://github.com/coiax/-tg-station/blob/these-ruins-are-heavy/code/modules/awaymissions/zlevel.dm#L92?

If not you could feasbly replace the suffix var with this